python/ray/rllib/utils/sampler.py

```diff
@@ -239,7 +239,8 @@ def _env_runner(env, policy, num_local_steps, horizon, obs_filter):
             if policy.is_recurrent:
                 features = pi_info["features"]
                 del pi_info["features"]
-            observation, reward, terminal, info = env.step(action)
+            observation, reward, terminal, info = env.step(
+                action.reshape(env.action_space.shape))
```

Comment:
This is a little fishy, the need to reshape means you're returning actions with the wrong dimensionality.

Reply:
I agree, I think the best thing to do is change the outputs of the networks so they handle the same shapes as the tensorflow version. Are there any output shapes that aren't supported by the tf version?